src/main/java/net/pms/dlna/RootFolder.java

```diff
@@ -94,29 +95,24 @@ public boolean isValid() {
 		return true;
 	}
 
-	@Override
 	public void discoverChildren() {
-		discoverChildren(true);
-	}
-
-	public void discoverChildren(boolean isAddGlobally) {
 		if (isDiscovered()) {
 			return;
 		}
 
 		if (configuration.isShowMediaLibraryFolder()) {
 			DLNAResource libraryRes = PMS.get().getLibrary();
 			if (libraryRes != null) {
-				addChild(libraryRes, true, isAddGlobally);
+				addChild(libraryRes);
```

Comment:
@valib why remove the `isAddGlobally`? It is what prevents the startup scan from taking up memory that is never released

Reply:
@SubJunk because your merging stopped UMS working not only on my TV but also on the web interface. The virtual folders were missing.

Reply:
Ah ok, I will need to re-add those and resolve it a different way, because this is an important thing that stops the program crashing on startup. I'm working on this now :)

Reply:
Which virtual folders were missing? Are they working now after my latest commit?